data_updates/Python/csv/one.csv

```diff
@@ -0,0 +1,5 @@
+etea
+dgesdg
+agesdg
+segrsagre
+aegrgr
```

Comment:
What's the purpose of this file? It's not proper CSV & isn't being used or generated anywhere.

Reply:
I think this was sometime @wakibi did for testing but can delete?